src/components/ConfirmationDialog.tsx

```
@@ -0,0 +1,146 @@
+            </ModalHeader>
+
+            <ModalBody py={4}>
+              <Text fontSize="md" mb={4} color="gray.600">
+                {consentText}
+              </Text>
+              <VStack spacing={3}>
+                {loading ? (
+                  <Spinner color="blue.600" />
+                ) : (
+                  documents.map((document) => (
+                    <UnorderedList mt={4}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)